motion_tracker/utils/crop_vis.py

```diff
@@ -1,7 +1,7 @@
 import cv2
-from motion_tracker.utils.image_cropping import Coords, crop_and_resize
+from image_cropping import Coords, crop_and_resize
 
```

Comment:
We need to figure something out on this.  It was working for me, and after this change it no longer runs for me (unless I run things from the utils directory.)

Can you add the repository's root directory to your pythonpath and see if the previous import works?